### ISSUE TITLE
[Request]: Keyboard shortcuts to enter fullscreen mode and to control zoom in an image view

### ISSUE THREAD

**Issue by nekohayo** [GeopJr/Tuba#639]
### Describe the request

When you click on an image in a toot, the lightbox view shows up. It has a few missing keyboard accessibility features:

* There is a toolbar button to enter/exit fullscreen mode, but it cannot be activated with `F11`  and `F` (would be good to support both, like in various other apps).  
~~I see that there's the `Enter` keyboard shortcut, but it's kind of non-standard/unexpected compared to other apps, so I'd suggest keeping it around as a 3rd keyboard shortcut, but considering F11 and F to be the primary ones? In fact, I discovered the `Enter` shortcut totally by chance, and I see now that it was not documented in the keyboard shortcuts dialog anyway...~~
* Zoom controls cannot be activated by the keyboard (and numpad's) `+`/`-` keys.
* The meatballs menu cannot be activated through `F10`

### Implementation Details

- [X] This follows the [GNOME HIG](https://developer.gnome.org/hig/).

**Comment by GeopJr** [comment]
> Zoom controls cannot be activated by the keyboard (and numpad's) +/- keys.
The shortcut is <kbd>Ctrl</kbd><kbd>+</kbd>/<kbd>Ctrl</kbd><kbd>-</kbd> (should be added to the shortcuts window if not already)

> I see that there's the Enter keyboard shortcut
It's not. Were you focused on it by any chance?

**Comment by nekohayo** [comment]
> The shortcut is Ctrl+/Ctrl- (should be added to the shortcuts window if not already)

Weird, I think I had tried that but it was not working for some reason; it works when trying again now, but indeed it was not documented in the shortcuts dialog :)


> >   I see that there's the Enter keyboard shortcut
>
> It's not. Were you focused on it by any chance?

Oh yes, I must have been focused on the fullscreen toolbar button and the Enter key hit that button because of that.

**Comment by GeopJr** [comment]
- Added the media viewer shortcuts and gestures to the list
- F11 should now toggle fullscreen
- F10 should now open the menu

What I haven't done:
- `F` for toggling fullscreen - due to the way media viewer works on the main branch I can't set a binding action for now and can't deal with translating keypress events
- `-` and `+` for zooming - I think the current shortcuts are more appropriate since they match the browser behavior

![Screenshot from 2023-11-16 15-29-32](https://github.com/GeopJr/Tuba/assets/18014039/d69cd7ba-140a-4788-aaed-ab3eb89898c5)

**Comment by nekohayo** [comment]
> `F` for toggling fullscreen - due to the way media viewer works on the main branch I can't set a binding action for now and can't deal with translating keypress events

I see in the diff of that MR that it was done with:

```xml
<property name="accelerator">F11</property>
```

Is it not possible to slap the `F` key along it just like this? I would've thought maybe this could work: 

```xml
<property name="accelerator">F11</property>
<property name="accelerator">f</property>
```

FWIW, grepping in Nautilus for `F5`, where they also map `Ctrl+R`, I see:

```
src/nautilus-window.c:
    nautilus_application_set_accelerators (app, "win.reload", ACCELS ("F5", "<ctrl>r", "Refresh", "Reload"));

src/resources/gtk/help-overlay.ui:
   <property name="accelerator">F5 &lt;Primary&gt;R</property>
```
